Code/GraphMol/FileParsers/MaeMolSupplier.cpp

```diff
@@ -10,243 +10,424 @@
 #include <iostream>
 #include <fstream>
 #include <map>
+
 #include <RDGeneral/BadFileException.h>
 #include <RDGeneral/FileParseException.h>
 #include <GraphMol/MolInterchange/details.h>
 #include <GraphMol/MolOps.h>
 #include <GraphMol/MonomerInfo.h>
 #include <GraphMol/RWMol.h>
 #include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/FileParsers/FileParserUtils.h>
+
+#include <boost/tokenizer.hpp>
+
+#include <maeparser/MaeConstants.hpp>
 #include <maeparser/Reader.hpp>
 
-using std::shared_ptr;
-using namespace schrodinger::mae;
+using namespace schrodinger;
+using RDKit::MolInterchange::bolookup;
 
 namespace RDKit {
 
-using RDKit::MolInterchange::bolookup;
+namespace {
 
-MaeMolSupplier::MaeMolSupplier(shared_ptr<std::istream> inStream,
-                               bool sanitize, bool removeHs) {
-  PRECONDITION(inStream, "bad stream");
-  dp_sInStream = inStream;
-  dp_inStream = inStream.get();
-  df_owner = true;
-  df_sanitize = sanitize;
-  df_removeHs = removeHs;
+const std::string PDB_ATOM_NAME = "s_m_pdb_atom_name";
+const std::string PDB_RESIDUE_NAME = "s_m_pdb_residue_name";
+const std::string PDB_CHAIN_NAME = "s_m_chain_name";
+const std::string PDB_INSERTION_CODE = "s_m_insertion_code";
+const std::string PDB_RESIDUE_NUMBER = "i_m_residue_number";
+const std::string PDB_OCCUPANCY = "r_m_pdb_occupancy";
+const std::string PDB_TFACTOR = "r_m_pdb_tfactor";
 
-  d_reader.reset(new Reader(dp_sInStream));
-  d_next_struct = d_reader->next("f_m_ct");
-}
+class PDBInfo {
```

Comment:
I'm a bit confused by this diff - this part seems like it's already merged in. I'm just trying to separate out the two fixes in my head.

Reply:
Ah, I think I know what the problem is: I created the PR, and after that, the `PDBInfo` was added, and I had to update the PR to include it too. Is it possible that it is that?